apps/extension/src/ui/domains/Sign/Qr/ScanQr.tsx

```diff
@@ -109,30 +110,38 @@ const Scanner = ({
   const inputMenu = useRef(null)
   useClickAway(inputMenu, () => setShowInputMenu(false))
 
+  const codeReader = useAtomValue(codeReaderAtom)
+
   useEffect(() => {
+    if (!codeReader) return
     if (!preview.current) return
 
     const aborted = new AbortController()
-
-    const codeReader = new BrowserQRCodeReader()
```

Comment:
No need to create a new one of these each time we run this effect, we can just use the one from the `codeReaderAtom`